hips/utils/wcs.py

```diff
@@ -78,35 +84,38 @@ def pix_to_sky(self, x, y) -> SkyCoord:
         return pixel_to_skycoord(x, y, self.wcs, self.WCS_ORIGIN_DEFAULT)
 
     @classmethod
-    def create(cls, skydir: SkyCoord, shape: tuple, coordsys: str = 'CEL',
+    def create(cls, skydir: SkyCoord, height: int, width: int, coordsys: str = 'icrs',
                projection: str = 'AIT', cdelt: float = 1.0, crpix: tuple = (1., 1.)) -> 'WCSGeometry':
         """Create WCS object programmatically (`WCSGeometry`).
 
         Parameters
         ----------
         skydir : `~astropy.coordinates.SkyCoord`
             Sky coordinate of the WCS reference point
-        shape : `tuple`
-            Shape of the image (Numpy axis order: y, x)
-        coordsys : `str`
+        height : `int`
+            Height of the image
+        width : `int`
+            Width of the image
```

Comment:
Mention "in pixels" or "number of pixel" to make it perfectly clear that this is not width e.g. as an angle on the sky.